lib/HTTP/Status.pm

```diff
@@ -262,6 +264,12 @@ that it has erred or is incapable of performing the request.
 
 This function is B<not> exported by default.
 
+=item status_code
+
+Returns a hash mapping numerical HTTP status codes (e.g. 200) to text status messages (e.g. "OK")
+
+This function is B<not> exported.
```

Comment:
What's the reasoning for not making this function exportable?

Reply:
Yeah, I don't remember why I did so now. Anyway, I adjusted.